css/style.css

```diff
@@ -0,0 +1,235 @@
+  background-color: #f1eee1;
+}
+
+.header-container{
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  width: 1024px;
+  background-color: #fffbe6;
+  color: #19282B;
+}
+
+.header-label{
+  flex:1;
+  display: flex;
+  justify-content: flex-end;
+  padding-right: 4%;
+  width: 1024px;
+}
+
+.header-label h1{
+  font-size: 2.5em;
+}
+
+/*Navigation*/
+.navbar{
+  display: flex;
+  flex: 1;
+  padding-left: 0.3em;
+  padding-right: 0.3em;
+  width: 1024px;
+  background-color: #327185;
+}
+
+.top-menu-list{
+  display: flex;
+  flex: 2;
+  justify-content: center;
+  align-items: center;
+  padding-left: 0px;
+  margin-top: 1em;
+  margin-left: 1.2em;
+  list-style: none;
+}
+
+.top-menu-list li{
+  display: inline;
+  padding: .4% .8% .4% .8%;
+  border-right: 1px solid black;
+  border-left: 1px solid black;
+}
+
+.top-menu-list li a{
+  color: white;
+  font-size: 1.8em;
+}
+
+.top-menu-list a:hover{
+  color: #E7B13C;
+}
+
+.top-menu-list li:nth-child(1){
+  border-left: 0 solid black;
+}
+.top-menu-list li:nth-child(5){
+  border-right: 0 solid black;
+}
+
+/*End Navigation*/
+/* Content about-me*/
+
+.main-container{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  width: 1024px;
+  background-color:  #fffbe6 ;
+}
+.wrapper-about-me{
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  padding:3% 4% 5% 4%;  
+  border-bottom: 2px solid gray;
+}
+.content-about-me {
+  display: flex;
+  flex:2;
+  flex-direction: column;
+  margin-right: 3em;
+}
+.content-about-me h2{
+  padding-bottom: 1%;
+  font-size: 2.4em;
+  border-bottom: 5px solid #EBCFA8;
+}
+
+.content-about-me p{
+  padding-top: 3%;
+  font-size: 1.6em;
+}
+
+.photo{
+  flex:.3;
+  display: flex;
+}
+.photo-description{
+  font-size: 1.4em;
+  text-align: center;
+  font-style: italic;
+}
+
+/* End Content about-me*/
+/* Content education*/
+
+.wrapper-education{
+  display: flex;
+  justify-content: center;
+  padding:3% 4% 5% 4%; 
+  width: 1024px;
+  border-bottom: 2px solid gray;
+}
+
+.content-education{
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 1024px;
+  font-size: 1.6em;
+}
+.content-education h2{
+  margin-bottom: 4%;
+  padding-bottom: 1%;
+  font-size: 1.6em;
+  border-bottom: 5px solid #EBCFA8;
+}
+.content-education h4{
+  margin-top: 1%;
+  margin-bottom: 1%;
+}
+/* End Content education*/
+/* content-skills*/
+
+.wrapper-skills{
+  display: flex;
+  justify-content: center;
+  padding:3% 4% 5% 4%; 
+  width: 1024px;
+  border-bottom: 2px solid gray;
+}
+.content-skills{
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  width: 1024px;
+  font-size: 1.6em;
+}
+
+.skills{
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  padding-right: 4%;
+}
+
+.experience{
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  padding-left: 4%;
+}
+.list-slills li{
+  padding: 3px;
```

Comment:
The closing brace is written on a new line and without an indent. The next rule after this is separated by an empty line. See more in this guide: http://codeguide.academy/html-css.html#html-language